Objects/stringlib/fastsearch.h

```diff
@@ -86,34 +102,46 @@ STRINGLIB(rfind_char)(const STRINGLIB_CHAR* s, Py_ssize_t n, STRINGLIB_CHAR ch)
        it doesn't seem as optimized as memchr(), but is still quite
        faster than our hand-written loop below */
 
-    if (n > 10) {
+    if (n > MEMCHR_CUT_OFF) {
 #if STRINGLIB_SIZEOF_CHAR == 1
-        p = memrchr(s, ch, n);
+        p = memrchr(s, ch, n * sizeof(STRINGLIB_CHAR));
```

Comment:
Ah, no good reason. Perhaps I tried to unify this with  the code for STRINGLIB_SIZEOF_CHAR != 1.